src/main/java/com/systelab/seed/features/patient/service/command/PatientUpdateCommandService.java

```diff
@@ -0,0 +1,23 @@
+package com.systelab.seed.features.patient.service.command;
+
+import com.systelab.seed.features.patient.model.Patient;
+import com.systelab.seed.features.patient.repository.PatientRepository;
+import com.systelab.seed.features.patient.service.PatientNotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class PatientUpdateCommandService {
+
+    private final PatientRepository patientRepository;
+
+    public Patient updatePatient(UUID id, Patient p) {
+        return this.patientRepository.findById(id).map(existing -> {
```

Comment:
I would suggest using the PatientQueryService to avoid repeating the same logic of findById + exception on each command service.

Reply:
Done